src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -508,6 +506,10 @@ class ADIOSWriter : public ILightweightPlugin
             mpi_pos = gc.getPosition();
             mpi_size = gc.getGpuNodes();
 
+            /* if number of aggregators is not set we use all mpi process as aggregator*/
+            if( mThreadParams.adiosAggregators == 0 )
+               mThreadParams.adiosAggregators=mpi_size.productOfComponents();
```

Comment:
well, but our # of OSTs is still pretty small compared to the writing processes (GPU: 32 to 60 per island, ~120 total). For the laser partition it's even harder.

I forwarded the question in our specific case to the adios dev's but I think one aggregator per node (4 to 8 GPUs respectively) will probably result in a better write performance, but we can check that.